gammapy/maps/wcs.py

```diff
@@ -882,6 +882,20 @@ def __repr__(self):
         )
         return str_
 
+    def __eq__(self, other):
+        """Test equality between two `~gammapy.maps.WcsGeom`. Espect tolerance of 1e-6 for `~astropy.wcs.WCS` objects"""
```

Comment:
Comment line is too long. IMO "Test equality between two `~gammapy.maps.WcsGeom`." doesn't add value and could be removed. The commend about WCS and 1e6 I would put inline above the line where you call the corresponding method.

Reply:
Done